lib/src/scanner.dart

```diff
@@ -237,7 +239,8 @@ class NgScanner {
       _state = NgScannerState.scanStart;
       return new NgToken.interpolationEnd(_current.offset);
     }
-    return handleError();
+    var overrideMessage = 'Unclosed mustache';
+    return handleError(overrideMessage: overrideMessage);
```

Comment:
Good catch! I think for this scenario, I'm going to submit another PR after this that gets rid of string passing and instead use error codes. This should handle the below issue as well :)